homeassistant/components/renault/device_tracker.py

```diff
@@ -42,16 +57,12 @@ def longitude(self) -> float | None:
         """Return longitude value of the device."""
         return self.coordinator.data.gpsLongitude if self.coordinator.data else None
 
-    @property
-    def source_type(self) -> SourceType:
-        """Return the source type of the device."""
-        return SourceType.GPS
-
 
-DEVICE_TRACKER_TYPES: tuple[RenaultDataEntityDescription, ...] = (
-    RenaultDataEntityDescription(
+DEVICE_TRACKER_TYPES: tuple[RenaultTrackerEntityDescription, ...] = (
+    RenaultTrackerEntityDescription(
         key="location",
         coordinator="location",
         translation_key="location",
+        source_type=SourceType.GPS,
```

Comment:
Will any of these tracker descriptions set a different source type? We could move it to a default.

Reply:
> Will any of these tracker descriptions set a different source type? We could move it to a default.

I've set as default on the `TrackerEntityDescription`